allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -0,0 +1,197 @@
+import logging
+import warnings
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from pynwb import NWBFile, TimeSeries
+
+from allensdk.brain_observatory import sync_utilities
+from allensdk.brain_observatory.behavior.data_files import SyncFile
+from allensdk.brain_observatory.behavior.data_files.eye_tracking_file import \
+    EyeTrackingFile
+from allensdk.brain_observatory.behavior.data_objects import DataObject
+from allensdk.brain_observatory.behavior.data_objects.base \
+    .readable_interfaces import \
+    NwbReadableInterface, DataFileReadableInterface
+from allensdk.brain_observatory.behavior.data_objects.base \
+    .writable_interfaces import \
+    NwbWritableInterface
+from allensdk.brain_observatory.behavior.eye_tracking_processing import \
+    process_eye_tracking_data, determine_outliers, determine_likely_blinks
+from allensdk.brain_observatory.nwb.eye_tracking.ndx_ellipse_eye_tracking \
+    import \
+    EllipseSeries, EllipseEyeTracking
+from allensdk.brain_observatory.sync_dataset import Dataset
+
+
+class EyeTrackingTable(DataObject, DataFileReadableInterface,
+                       NwbReadableInterface, NwbWritableInterface):
+    """corneal, eye, and pupil ellipse fit data"""
+    _logger = logging.getLogger(__name__)
+
+    def __init__(self, eye_tracking: pd.DataFrame):
+        super().__init__(name='eye_tracking', value=eye_tracking)
+
+    def to_nwb(self, nwbfile: NWBFile) -> NWBFile:
```

Comment:
docstring?

Reply:
If you feel that docstrings are not necessary, maybe we can discuss with @danielsf and see if we can come to an agreement.